### PR TITLE
LLREST: Wrap at 140 columns

### PR DESCRIPTION
**Pull request by nik9000**
Wraps a single line in a test to fit on 140 characters, removing the
only checkstyle suppression for the clients directory.

### CODE REVIEW

File: client/rest/src/test/java/org/elasticsearch/client/HeapBufferedAsyncResponseConsumerTests.java [hunk 1]
```diff
@@ -106,7 +106,8 @@ public void testCanConfigureHeapBufferLimitFromOutsidePackage() throws ClassNotF
             IllegalAccessException, InvocationTargetException, InstantiationException {
         int bufferLimit = randomIntBetween(1, Integer.MAX_VALUE);
         //we use reflection to make sure that the class can be instantiated from the outside, and the constructor is public
-        Constructor<?> constructor = HttpAsyncResponseConsumerFactory.HeapBufferedResponseConsumerFactory.class.getConstructor(Integer.TYPE);
+        Constructor<?> constructor = HttpAsyncResponseConsumerFactory.HeapBufferedResponseConsumerFactory.class
+                .getConstructor(Integer.TYPE);
```
Comment (jasontedor) [inline client/rest/src/test/java/org/elasticsearch/client/HeapBufferedAsyncResponseConsumerTests.java]: Does it fit if you wrap after the equals?

**Review by jasontedor — COMMENTED** [review]
I left a question.

File: client/rest/src/test/java/org/elasticsearch/client/HeapBufferedAsyncResponseConsumerTests.java (hunk 1 above)
Comment (nik9000) [reply client/rest/src/test/java/org/elasticsearch/client/HeapBufferedAsyncResponseConsumerTests.java]: Sure! I'll switch it.

**Review by jasontedor — APPROVED** [review]
LGTM.

**Comment by nik9000** [comment]
@elasticmachine, retest this please

**Comment by nik9000** [comment]
Thanks for reviewing @jasontedor!

**Pull request merged**